src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { FontType } from 'styled-components'
+import { BaseButton } from './styles/Button.styled'
+
+type Props = {
+  text: string
+  hasActive?: boolean
+  link?: string | undefined
+  htmlType?: 'button' | 'submit' | 'reset' | undefined
+  fontType: FontType
+}
+
+const defaultProps = {
+  hasActive: false,
+  link: null,
+  htmlType: 'submit',
```

Comment:
default might want to be button or undefined instead?

Reply:
Resolved in commit 8cb1f2e3fa2ff50e16fa4367e6681ec6e97021c9.